packages/dd-trace/test/setup/services/aws_sdk.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
```

Comment:
File is named `aws_sdk` but service is named `localstack`. They should match to make it easier to understand.

Reply:
makes sense, updated. frankly i'm not entirely clear where this file is getting run? It's used to validate that the container for these tests is listening on the right host/port? i used the gcloud-pub-sub instrumentation skeleton as a guide here but not sure i fully grok the use case